samples/ChangeFeedMigrationTool/Program.cs

```diff
@@ -0,0 +1,161 @@
+        private string monitoredUri = ConfigurationManager.AppSettings["monitoredUri"];
+        private string monitoredSecretKey = ConfigurationManager.AppSettings["monitoredSecretKey"];
+        private string monitoredDbName = ConfigurationManager.AppSettings["monitoredDbName"];
+        private string monitoredCollectionName = ConfigurationManager.AppSettings["monitoredCollectionName"];
+        private int monitoredThroughput = int.Parse(ConfigurationManager.AppSettings["monitoredThroughput"]);
+
+        // optional setting to store lease collection on different account
+        // set lease Uri, secretKey and DbName to same as monitored if both collections 
+        // are on the same account
+        private string leaseUri = ConfigurationManager.AppSettings["leaseUri"];
+        private string leaseSecretKey = ConfigurationManager.AppSettings["leaseSecretKey"];
+        private string leaseDbName = ConfigurationManager.AppSettings["leaseDbName"];
+        private string leaseCollectionName = ConfigurationManager.AppSettings["leaseCollectionName"];
+        private int leaseThroughput = int.Parse(ConfigurationManager.AppSettings["leaseThroughput"]);
+
+        // destination collection for data movement in this sample  
+        // could be same or different account 
+        private string destUri = ConfigurationManager.AppSettings["destUri"];
+        private string destSecretKey = ConfigurationManager.AppSettings["destSecretKey"];
+        private string destDbName = ConfigurationManager.AppSettings["destDbName"];
+        private string destCollectionName = ConfigurationManager.AppSettings["destCollectionName"];
+        private int destThroughput = int.Parse(ConfigurationManager.AppSettings["destThroughput"]);
+
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Change Feed Migration Sample");
+            Program newApp = new Program();
+
+            // create collections
+            newApp.CreateCollectionAsync(
+                newApp.monitoredUri, 
+                newApp.monitoredSecretKey, 
+                newApp.monitoredDbName, 
+                newApp.monitoredCollectionName, 
+                newApp.monitoredThroughput).Wait();
+
+            newApp.CreateCollectionAsync(
+                newApp.leaseUri, 
+                newApp.leaseSecretKey, 
+                newApp.leaseDbName, 
+                newApp.leaseCollectionName, 
+                newApp.leaseThroughput).Wait();
+
+            newApp.CreateCollectionAsync(
+                newApp.destUri, 
+                newApp.destSecretKey, 
+                newApp.destDbName, 
+                newApp.destCollectionName, 
+                newApp.destThroughput).Wait();
+
+            // run change feed processor
+            newApp.RunChangeFeedHostAsync().Wait(); 
+        }
+
+        public async Task CreateCollectionAsync(string newUri, string secretKey, string dbName, string collectionName, int throughput)
```

Comment:
This is not really CreateCollectionAsync, it's CreateCollectionIfNotExistsAsync or MakeSureCollectionExistsAsync.